recipes/default.rb

```diff
@@ -39,6 +39,10 @@
       plugin true
     end
   end
+  execute "plugin restart" do
```

Comment:
This resource declaration is totally meaningless. If you'd like to notify service restart after plugin installation, please move "notifies" to td_agent_gem.

Reply:
The issue is that the agent needs to be restarted after the plugins are installed -- which this adjustment does. I'm not sure where you mean by the "td_agent_gem". Could you clarify?

Reply:
After digging, I think you mean to put it in the code blocks like this one:
`    td_agent_gem plugin do
      plugin true`
I will need test this further next week

Reply:
the simplified change works correctly